pywt/_cwt.py

```diff
@@ -34,7 +34,8 @@ def next_fast_len(n):
             return 2**ceil(np.log2(n))
 
 
-def cwt(data, scales, wavelet, sampling_period=1., method='conv', axis=-1):
+def cwt(data, scales, wavelet, sampling_period=1., method='conv', 
+        precision=12, axis=-1):
```

Comment:
```suggestion
def cwt(data, scales, wavelet, sampling_period=1., method='conv', 
        axis=-1, *, precision=12):
```

`axis` needs to remain at its current position so we don't break any user code that may have been providing the arguments positionally.

Now that we don't support Python 2.x anymore, the new argument can also be keyword-only.